core/src/consensus.rs

```diff
@@ -932,9 +983,9 @@ impl Tower {
             self.lockouts = vote_state;
             self.do_initialize_lockouts(root, |v| v.slot > root);
             trace!(
-                "{} lockouts initialized to {:?}",
+                "Lockouts in tower for {} is initialized using bank {}",
                 self.node_pubkey,
-                self.lockouts
```

Comment:
Currently, we are `{:?}`-ing a whole Tower once at validator startup later with `info!` to ease any persistent tower related debuggng.